python/cuml/svm/svm_base.pyx

```diff
@@ -281,8 +281,8 @@ class SVMBase(Base):
             return self.gamma
 
     def _calc_coef(self):
-        return np.dot(self.dual_coef_.copy_to_host(),
-                      self.support_vectors_.copy_to_host())
+        return np.dot(self._dual_coef_.to_output('numpy'),
```

Comment:
Why `np.dot` and `to_output('numpy')`? Can't that run on GPU using cupy instead?

Reply:
I can make this change but for some reason change `__getattr__` and `__setattr__` to `cupy` causes a discrepancy in the predictions before and after pickling. I'm investigating it further.

Reply:
This is because the functions that are calling these expect a CPU version returned to them (especially when using external libraries). Which is why the original version was also using `copy_to_host`

Reply:
Oh ok, that makes sense!